demo/src/main/java/com/novoda/merlin/demo/presentation/base/MerlinActivity.java

```diff
@@ -50,6 +50,12 @@ protected void onStop() {
         Logger.detach(logHandle);
     }
 
+    @Override
+    protected void onDestroy() {
+        merlin = null;
```

Comment:
Yes, I think so. We need to clear out all the references to `context` that we currently stored internally.